src/components/atomic/CISBanner/CISBanner.jsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+
+import { useStateValue } from '../../../store/store';
+import './CISBanner.scss';
+
+const CISBanner = () => {
+	const [{ cisBannerImgUrlLarge, cisBannerImgUrlSmall }] = useStateValue();
+
+	const handleLiveHelpClick = (event) => {
```

Comment:
my thought with this is to turn the on click to a function that gets passed in so that it can be configured in frontendGlobals in case that url ever changes?

Reply:
That makes sense to me, I must have missed the discussion where we talked about passing in the function.
Does that mean I should include that in the App.js checks for whether all the required variables have been passed in? Or do we not want that to be required

Reply:
I don't think it needs to be required but the handleClick just calls that passed in function.  (easy enough for unit tests, too!)

Reply:
Updated! Please take a look and let me know if it's set up correctly!